.github/workflows/build.yml

```diff
@@ -12,14 +12,12 @@ jobs:
         uses: actions/setup-go@v1
         with:
           go-version: ${{ matrix.go }}
+
       - name: Check out source
         uses: actions/checkout@v1
       - name: Install Linters
         run: "curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0"
-      - name: Build
-        env:
-          GO111MODULE: "on"
-        run: go build -race ./...
```

Comment:
Good addition.  Funny we weren't doing it already.  Although, does `-race` in the build command add anything since the outputs aren't run?

Reply:
I don't think `-race` does anything, but I saw it here, so I wasn't sure. Removed `run_tests.sh` in latest commit.